@ashharrison90 Indeed

```
❯ yarn typecheck
public/app/features/alerting/unified/components/Provisioning.tsx:44:71 - error TS2322: Type '{ children: (string | { provenance: string | undefined; })[]; i18nKey: string; provenance: string | undefined; }' is not assignable to type 'IntrinsicAttributes & TransProps'.
  Property 'provenance' does not exist on type 'IntrinsicAttributes & TransProps'.

44       <Trans i18nKey="alerting.provisioning.badge-tooltip-provenance" provenance={provenance}>
                                                                         ~~~~~~~~~~

public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx:149:86 - error TS2322: Type '{ children: (string | { evaluateEvery: string; })[]; i18nKey: string; evaluateEvery: string; }' is not assignable to type 'IntrinsicAttributes & TransProps'.
  Property 'evaluateEvery' does not exist on type 'IntrinsicAttributes & TransProps'.

149                     <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.text" evaluateEvery={evaluateEvery}>
                                                                                         ~~~~~~~~~~~~~

public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx:13:12 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; key: string; }' but required in type 'TransProps'.

13           <Trans key="transformations.empty.add-transformation-header">Start transforming data</Trans>
              ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx:16:12 - error TS2741: Property 'i18nKey' is missing in type '{ children: (string | Element)[]; key: string; }' but required in type 'TransProps'.

16           <Trans key="transformations.empty.add-transformation-body">
              ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:81:18 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

81                 <Trans>Description</Trans>
                    ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:88:22 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

88                     <Trans>No description available</Trans>
                        ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:95:18 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

95                 <Trans>Type</Trans>
                    ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:101:20 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

101                   <Trans>Unknown</Trans>
                       ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:107:18 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

107                 <Trans>Unit</Trans>
                     ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:113:20 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

113                   <Trans>{unit}</Trans>
                       ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.

public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:119:18 - error TS2741: Property 'i18nKey' is missing in type '{ children: string; }' but required in type 'TransProps'.

119                 <Trans>Labels</Trans>
                     ~~~~~

  public/app/core/internationalization/index.tsx:64:3
    64   i18nKey: string;
         ~~~~~~~
    'i18nKey' is declared here.


Found 11 errors in 4 files.

Errors  Files
     1  public/app/features/alerting/unified/components/Provisioning.tsx:44
     1  public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx:149
     2  public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx:13
     7  public/app/features/trails/ActionTabs/MetricOverviewScene.tsx:81
```

--- a/public/app/features/alerting/unified/components/Provisioning.tsx
+++ b/public/app/features/alerting/unified/components/Provisioning.tsx
@@ -41,7 +41,7 @@ export const ProvisioningBadge = ({
 
   if (tooltip) {
     const provenanceTooltip = (
-      <Trans i18nKey="alerting.provisioning.badge-tooltip-provenance" provenance={provenance}>
+      <Trans i18nKey="alerting.provisioning.badge-tooltip-provenance" values={{ provenance }}>
         This resource has been provisioned via {{ provenance }} and cannot be edited through the UI
       </Trans>
     );

--- a/public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx
+++ b/public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx
@@ -146,7 +146,7 @@ function FolderGroupAndEvaluationInterval({
               <Stack direction="column" gap={1}>
                 {getValues('group') && getValues('evaluateEvery') && (
                   <span>
-                    <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.text" evaluateEvery={evaluateEvery}>
+                    <Trans i18nKey="alert-rule-form.evaluation-behaviour-group.text" values={{ evaluateEvery }}>
                       All rules in the selected group are evaluated every {{ evaluateEvery }}.
                     </Trans>
                     {!isNewGroup && (

--- a/public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx
+++ b/public/app/features/dashboard-scene/panel-edit/PanelDataPane/EmptyTransformationsMessage.tsx
@@ -10,10 +10,10 @@ export function EmptyTransformationsMessage(props: EmptyTransformationsProps) {
     <Box alignItems="center" padding={4}>
       <Stack direction="column" alignItems="center" gap={2}>
         <Text element="h3" textAlignment="center">
-          <Trans key="transformations.empty.add-transformation-header">Start transforming data</Trans>
+          <Trans i18nKey="transformations.empty.add-transformation-header">Start transforming data</Trans>
         </Text>
         <Text element="p" textAlignment="center" data-testid={selectors.components.Transforms.noTransformationsMessage}>
-          <Trans key="transformations.empty.add-transformation-body">
+          <Trans i18nKey="transformations.empty.add-transformation-body">
             Transformations allow data to be changed in various ways before your visualization is shown.
             <br />
             This includes joining data together, renaming fields, making calculations, formatting data for display, and

--- a/public/app/features/trails/ActionTabs/MetricOverviewScene.tsx
+++ b/public/app/features/trails/ActionTabs/MetricOverviewScene.tsx
@@ -78,45 +78,39 @@ export class MetricOverviewScene extends SceneObjectBase<MetricOverviewSceneStat
           <>
             <Stack direction="column" gap={0.5}>
               <Text weight={'medium'}>
-                <Trans>Description</Trans>
+                <Trans i18nKey="trails.metric-overview.description-label">Description</Trans>
               </Text>
               <div style={{ maxWidth: 360 }}>
                 {metadata?.help ? (
                   <div>{metadata?.help}</div>
                 ) : (
                   <i>
-                    <Trans>No description available</Trans>
+                    <Trans i18nKey="trails.metric-overview.no-description">No description available</Trans>
                   </i>
                 )}
               </div>
             </Stack>
             <Stack direction="column" gap={0.5}>
               <Text weight={'medium'}>
-                <Trans>Type</Trans>
+                <Trans i18nKey="trails.metric-overview.type-label">Type</Trans>
               </Text>
               {metadata?.type ? (
                 <div>{metadata?.type}</div>
               ) : (
                 <i>
-                  <Trans>Unknown</Trans>
+                  <Trans i18nKey="trails.metric-overview.unknown-type">Unknown</Trans>
                 </i>
               )}
             </Stack>
             <Stack direction="column" gap={0.5}>
               <Text weight={'medium'}>
-                <Trans>Unit</Trans>
+                <Trans i18nKey="trails.metric-overview.unit-label">Unit</Trans>
               </Text>
-              {metadata?.unit ? (
-                <div>{metadata?.unit}</div>
-              ) : (
-                <i>
-                  <Trans>{unit}</Trans>
-                </i>
-              )}
+              {metadata?.unit ? <div>{metadata?.unit}</div> : <i>{unit}</i>}
             </Stack>
             <Stack direction="column" gap={0.5}>
               <Text weight={'medium'}>
-                <Trans>Labels</Trans>
+                <Trans i18nKey="trails.metric-overview.labels-label">Labels</Trans>
               </Text>
               {labelOptions.length === 0 && 'Unable to fetch labels.'}
               {labelOptions.map((l) => (

--- a/public/locales/en-US/grafana.json
+++ b/public/locales/en-US/grafana.json
@@ -1,6 +1,5 @@
 {
   "_comment": "The code is the source of truth for English phrases. They should be updated in the components directly, and additional plurals specified in this file.",
-  "{unit}": "{unit}",
   "access-control": {
     "add-permission": {
       "role-label": "Role",
@@ -669,7 +668,6 @@
       "message": "No data sources found"
     }
   },
-  "Description": "Description",
   "explore": {
     "add-to-dashboard": "Add to dashboard",
     "add-to-library-modal": {
@@ -940,7 +938,6 @@
       "refresh": "Refresh"
     }
   },
-  "Labels": "Labels",
   "library-panel": {
     "add-modal": {
       "cancel": "Cancel",
@@ -1541,7 +1538,6 @@
     },
     "title": "Latest from the blog"
   },
-  "No description available": "No description available",
   "notifications": {
     "empty-state": {
       "description": "Notifications you have received will appear here",
@@ -2233,15 +2229,22 @@
       "select-search-input": "Type to search (country, city, abbreviation)"
     }
   },
+  "trails": {
+    "metric-overview": {
+      "description-label": "Description",
+      "labels-label": "Labels",
+      "no-description": "No description available",
+      "type-label": "Type",
+      "unit-label": "Unit",
+      "unknown-type": "Unknown"
+    }
+  },
   "transformations": {
     "empty": {
       "add-transformation-body": "Transformations allow data to be changed in various ways before your visualization is shown.<1></1>This includes joining data together, renaming fields, making calculations, formatting data for display, and more.",
       "add-transformation-header": "Start transforming data"
     }
   },
-  "Type": "Type",
-  "Unit": "Unit",
-  "Unknown": "Unknown",
   "user-orgs": {
     "current-org-button": "Current",
     "name-column": "Name",

--- a/public/locales/pseudo-LOCALE/grafana.json
+++ b/public/locales/pseudo-LOCALE/grafana.json
@@ -1,6 +1,5 @@
 {
   "_comment": "Ŧĥę čőđę įş ŧĥę şőūřčę őƒ ŧřūŧĥ ƒőř Ēŉģľįşĥ pĥřäşęş. Ŧĥęy şĥőūľđ þę ūpđäŧęđ įŉ ŧĥę čőmpőŉęŉŧş đįřęčŧľy, äŉđ äđđįŧįőŉäľ pľūřäľş şpęčįƒįęđ įŉ ŧĥįş ƒįľę.",
-  "{unit}": "{ūŉįŧ}",
   "access-control": {
     "add-permission": {
       "role-label": "Ŗőľę",
@@ -669,7 +668,6 @@
       "message": "Ńő đäŧä şőūřčęş ƒőūŉđ"
     }
   },
-  "Description": "Đęşčřįpŧįőŉ",
   "explore": {
     "add-to-dashboard": "Åđđ ŧő đäşĥþőäřđ",
     "add-to-library-modal": {
@@ -940,7 +938,6 @@
       "refresh": "Ŗęƒřęşĥ"
     }
   },
-  "Labels": "Ŀäþęľş",
   "library-panel": {
     "add-modal": {
       "cancel": "Cäŉčęľ",
@@ -1541,7 +1538,6 @@
     },
     "title": "Ŀäŧęşŧ ƒřőm ŧĥę þľőģ"
   },
-  "No description available": "Ńő đęşčřįpŧįőŉ äväįľäþľę",
   "notifications": {
     "empty-state": {
       "description": "Ńőŧįƒįčäŧįőŉş yőū ĥävę řęčęįvęđ ŵįľľ äppęäř ĥęřę",
@@ -2233,15 +2229,22 @@
       "select-search-input": "Ŧypę ŧő şęäřčĥ (čőūŉŧřy, čįŧy, äþþřęvįäŧįőŉ)"
     }
   },
+  "trails": {
+    "metric-overview": {
+      "description-label": "Đęşčřįpŧįőŉ",
+      "labels-label": "Ŀäþęľş",
+      "no-description": "Ńő đęşčřįpŧįőŉ äväįľäþľę",
+      "type-label": "Ŧypę",
+      "unit-label": "Ůŉįŧ",
+      "unknown-type": "Ůŉĸŉőŵŉ"
+    }
+  },
   "transformations": {
     "empty": {
       "add-transformation-body": "Ŧřäŉşƒőřmäŧįőŉş äľľőŵ đäŧä ŧő þę čĥäŉģęđ įŉ väřįőūş ŵäyş þęƒőřę yőūř vįşūäľįžäŧįőŉ įş şĥőŵŉ.<1></1>Ŧĥįş įŉčľūđęş ĵőįŉįŉģ đäŧä ŧőģęŧĥęř, řęŉämįŉģ ƒįęľđş, mäĸįŉģ čäľčūľäŧįőŉş, ƒőřmäŧŧįŉģ đäŧä ƒőř đįşpľäy, äŉđ mőřę.",
       "add-transformation-header": "Ŝŧäřŧ ŧřäŉşƒőřmįŉģ đäŧä"
     }
   },
-  "Type": "Ŧypę",
-  "Unit": "Ůŉįŧ",
-  "Unknown": "Ůŉĸŉőŵŉ",
   "user-orgs": {
     "current-org-button": "Cūřřęŉŧ",
     "name-column": "Ńämę",

